packages/kit/src/runtime/server/index.js

```diff
@@ -17,7 +17,7 @@ export async function respond(request, options, state) {
 
 	const normalized = normalize_path(url.pathname, options.trailing_slash);
 
-	if (normalized !== url.pathname) {
+	if (normalized !== url.pathname && !state.prerender?.fallback) {
```

Comment:
I forget where we're at with using optional chaining on the server. I recall there being issues in some hosting providers because of the JS features they supported. Are those now addressed and/or would this be getting transpiled (as either part of Kit's build, or during the adapt phase)?

Reply:
i think we're in the clear — we only support 14 and above: https://node.green/#ES2020-features-optional-chaining-operator------optional-property-access

i think we're using optional chaining in a ton of places already